GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -85,6 +85,10 @@ public override IEnumerable<EventListener> CreateTelemetryListeners(string provi
                 yield return etwListener;
             }
 
+            // TODO: enable the daemon-based telemetry listener once we're happy.
+            // See GitHub issue: https://github.com/Microsoft/VFSForGit/issues/739
+            yield break;
+

```

Comment:
While you're in here could you swap out the `var` on line 82 and 92 with an explicit type?